app/src/org/commcare/views/widgets/RecordingFragment.java

```diff
@@ -190,6 +192,7 @@ private void startRecording() {
         recorder.start();
         recordingDuration.setBase(SystemClock.elapsedRealtime());
         recordingInProgress();
+        Logger.log(LogTypes.TYPE_MEDIA_EVENT, "Recording started");

```

Comment:
can we log the scenario above when ` if (MediaUtil.isRecordingActive(getContext())) {` is true so that we know in logs that recording was not started due to this reason and not anything unknown.